.circleci/config.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/marquez
+    docker:
+      - image: circleci/openjdk:8-jdk
```

Comment:
should we standardize on java 8 for development?

Reply:
Actually, let's bump this to `java 9`. And are you thinking enforcing `java 9`?

Reply:
```java
if (JavaVersion.current() != JavaVersion.VERSION_1_9) {
  throw new GradleException("This build must be run with Java 9")
}
```

Reply:
I think some warning/enforcement is good just so that there are no surprises when the code builds locally but fails on the build server.